01_intro-r.qmd

```diff
@@ -54,68 +64,85 @@ R comes from the S programming language and S-PLUS system. In addition to offeri
 
 Source: "Stages in the Evolution of S" via [Roger Peng](https://simplystatistics.org/posts/2018-07-12-use-r-keynote-2018/)
```

Comment:
What should we do with source statements like this? Leave them in the body or move them to references?

Reply:
Probably references - I will work on this unless you have concerns

Reply:
Update - for anything where there is a citation to ideas shared by others, I add to the references which feels appropriate. Otherwise, I keep a link.

Reply:
Let me know if you have concerns!

Reply:
That sounds great!